### PR TITLE
AccessRequestDefinition and CredentialRequestDefinition explanation

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Adding information to create/update AccessRequestDefinition / CredentialRequestDefinition

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 568d862 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-325.amplify-central.opendocs-builder.pcloud.axway.int |

**Comment by cbordier** [comment]
@lbadenhop it is ready for your review.

File: content/en/docs/integrate_with_central/customize_ard_crd.md [hunk 1]
```diff
@@ -0,0 +1,627 @@
+---
+title: Customize access request and credentials request screens
+linkTitle: Customize access request and credentials request screens
+weight: 300
+---
+Amplify Allows to personalize the access request screen and the credential screen to ask any information to your consumer and send them in return appropriate answer
```
Comment (robertengaxway) [inline content/en/docs/integrate_with_central/customize_ard_crd.md:6]: Cyrille, I think there is a typo here.  are you trying to say?
"ask any information to your consumer and have the reply with an appropriate answer"

File: content/en/docs/integrate_with_central/customize_ard_crd.md [hunk 2]
```diff
@@ -0,0 +1,627 @@
...
+{
+    "group": "management",
+    "apiVersion": "v1alpha1",
+    "kind": "AccessRequestDefinition",
+    "name": "api-scopes",
+    "title": "API Scopes",
+    "metadata": {
+        "scope": {
+            "kind": "Environment",
+            "name": "environment-name",
```
Comment (robertengaxway) [inline content/en/docs/integrate_with_central/customize_ard_crd.md:261]: remove comma after "environment-name" ?

File: content/en/docs/integrate_with_central/customize_ard_crd.md [hunk 3]
```diff
@@ -0,0 +1,627 @@
...
+{
+    "group": "management",
+    "apiVersion": "v1alpha1",
+    "kind": "CredentialRequestDefinition",
+    "name": "name-for-credential-request-definition",
+    "title": "title-for-credential-request-definition",
+    "metadata": {
+        "scope": {
+            "kind": "Environment",
+            "name": "environment-name",
```
Comment (robertengaxway) [inline content/en/docs/integrate_with_central/customize_ard_crd.md:370]: remove comma?

File: content/en/docs/integrate_with_central/customize_ard_crd.md [hunk 4]
```diff
@@ -0,0 +1,627 @@
...
+{
+        "group": "management",
+        "apiVersion": "v1alpha1",
+        "kind": "CredentialRequestDefinition",
+        "name": "api-key",
+        "title": "API Key",
+        "metadata": {
+            "scope": {
+                "kind": "Environment",
+                "name": "environment-name",
```
Comment (robertengaxway) [inline content/en/docs/integrate_with_central/customize_ard_crd.md:410]: remove comma ?

File: content/en/docs/integrate_with_central/customize_ard_crd.md [hunk 5]
```diff
@@ -0,0 +1,627 @@
...
+{
+        "group": "management",
+        "apiVersion": "v1alpha1",
+        "kind": "CredentialRequestDefinition",
+        "name": "oauth-client-credential",
+        "title": "OAuth Client credential",
+        "metadata": {
+            "scope": {
+                "kind": "Environment",
+                "name": "environment-name",
```
Comment (robertengaxway) [inline content/en/docs/integrate_with_central/customize_ard_crd.md:474]: remove comma?

File: content/en/docs/integrate_with_central/customize_ard_crd.md [hunk 6]
```diff
@@ -0,0 +1,627 @@
...
+{
+        "group": "management",
+        "apiVersion": "v1alpha1",
+        "kind": "CredentialRequestDefinition",
+        "name": "oauth-client-certificate",
+        "title": "OAuth Client Certificate",
+        "metadata": {
+            "scope": {
+                "kind": "Environment",
+                "name": "environment-name",
```
Comment (robertengaxway) [inline content/en/docs/integrate_with_central/customize_ard_crd.md:527]: remove comma ?

File: content/en/docs/integrate_with_central/customize_ard_crd.md [hunk 7]
```diff
@@ -0,0 +1,627 @@
...
+{
+    "group": "management",
+    "apiVersion": "v1alpha1",
+    "kind": "APIServiceInstance",
+    "name": "api-service-instance",
+    "title": "api-service-instance-title",
+    "metadata": {
+        "scope": {
+            "kind": "Environment",
+            "name": "environment-name",
```
Comment (robertengaxway) [inline content/en/docs/integrate_with_central/customize_ard_crd.md:588]: remove comma?

**Review by robertengaxway — COMMENTED** [review]
content looks good, see comments about extra comma in json format

File: content/en/docs/integrate_with_central/customize_ard_crd.md [hunk 8]
```diff
@@ -0,0 +1,627 @@
...
+
+```json
+{
+  "properties": {
+    "firstName": {
+      "type": "string",
+      "title": "First name",
+      "description": "Enter your first name. Range between 3 and 25 characters",
+      "minLength": 3,
+      "maxLength": 25,
```
[the remark this answers is not in the document]
Comment (cbordier) [reply content/en/docs/integrate_with_central/customize_ard_crd.md:66]: I did test it without issues.

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**